.readthedocs.yml

```diff
@@ -1,4 +1,22 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
 version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3"
```

Comment:
I found in a recent project of mine that I needed these fields for the build to work. Python version `"3"` should just use the latest stable Python version. The rest of the additions in this file are from the [template](https://docs.readthedocs.io/en/stable/config-file/v2.html).